widgets/features/features.php

```diff
@@ -52,13 +52,13 @@ function get_widget_form(){
 
 					'container_color' => array(
 						'type' => 'color',
-						'label' => __('Container color', 'so-widgets-bundle'),
+						'label' => __('Icon Container Color', 'so-widgets-bundle'),
```

Comment:
To be consistent with other labels in this form this should us sentence case, i.e. _Icon container color_.